app/src/main/java/com/keylesspalace/tusky/components/account/list/ListsForAccountFragment.kt

```diff
@@ -60,46 +67,45 @@ class ListsForAccountFragment : DialogFragment(), Injectable {
         viewModel.setup(requireArguments().getString(ARG_ACCOUNT_ID)!!)
     }
 
-    override fun onStart() {
-        super.onStart()
-        dialog?.apply {
-            window?.setLayout(
-                LinearLayout.LayoutParams.MATCH_PARENT,
```

Comment:
Do we need this dialog to be full screen?
(In that case simply using a RelativeLayout is enough and also needed for the alert dialog buttons to be at the bottom.)

Having it small has the advantage that you can also tap anywhere outside to cancel.

(What happens anyway if the list is too long for full screen? There is no scrolling yet here?)

Reply:
If it was fullscreen it doesn't need to be a dialog, could just be a regular frament. I think it is better if it looks like a dialog, and yes of course it should scroll.